src/components/pages/style.scss

```diff
@@ -30,16 +39,47 @@
 	&__page-header {
 		height: 35px;
 		padding: 10px;
-		color: #555d66;
-		border-bottom: 1px solid #ccc;
 		display: flex;
 		align-items: center;
+		.theme-light & {
+			color: $dark-gray-700;
+			border-bottom: 1px solid $light-gray-100;
+		}
+		.theme-dark & {
+			color: $light-gray-200;
+			border-bottom: 1px solid $dark-gray-400;
+		}
 	}
 
 	&__page-title-logo {
 		margin: 5px;
 		height: 25px;
-		fill: #555d66;
+		.theme-light & {
+			fill: $blue-wordpress-700;
+		}
+		.theme-dark & {
+			fill: $blue-medium-100;
+		}
+	}
+
+	.components-icon-button .dashicon {
```

Comment:
probably also need hover styles for this button and the one below

Reply:
solved by reversing the colors!